clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/servergroup/EnableOpenstackAtomicOperation.groovy

```diff
@@ -65,8 +66,10 @@ class EnableOpenstackAtomicOperation extends AbstractEnableDisableOpenstackAtomi
       } as Supplier<LoadBalancerV2StatusTree>)]
     }
     CompletableFuture.allOf([loadBalancers.values(), statusTrees.values(), ips.values()].flatten() as CompletableFuture[]).join()
+    Map<String, BlockingStatusChecker> checkers = loadBalancerIds.collectEntries { [(it):createBlockingActiveStatusChecker(description.credentials, description.region, it)] }
```

Comment:
Makes sense ... Thought you were walking through the checker map instead of the instance list.  Please disregard.